css/style.css

```diff
@@ -680,15 +705,34 @@ button.button-inline:hover {
 
 .cell-board-title {
 	width: 50%;
+}
 
+#boardlist tr.deleted td * {
```

Comment:
why not just change the opacity of the whole row with `#boardlist tr.deleted {` ?